### ISSUE TITLE
DB_USER cannot be changed

### ISSUE THREAD

**Issue by jens9167** [uni-halle/etherpad-lite-docker#2]
When using the default value for ETHERPAD_DB_USER=root in combination with the provided root password for the database, the installation works fine.
If the user name for the database access specified in ETHERPAD_DB_USER is changed, access is denied 

```
etherpad_1  | mysqlshow: Access denied for user 'etheruser'@'172.24.0.2' (using password: YES)
mariadb_1   | 2021-02-10 12:27:22 14 [Warning] Access denied for user 'etheruser'@'172.24.0.2' (using password: YES)
etherpad_1  | Wed Feb 10 12:27:22 UTC 2021 - still trying connect to DBMS on
```

.env file (passwords have been changed)

```

# MySQL/ MariaDB connection parameters
MYSQL_HOST=localhost
MYSQL_ROOT_PASSWORD=k87gJn29gvE578gjVBHDB
MYSQL_USER=etheruser
MYSQL_PASSWORD=94156BD8hgBjdbjuV2yu

# Etherpad parameters
ETHERPAD_TITLE=Etherpad Lite
ETHERPAD_DEFAULT_PAD_TEXT=text.
ETHERPAD_HOST_PORT=8080
ETHERPAD_ADMIN_PASSWORD=jF2g2ndCiyhjhdiyh28h
ETHERPAD_ADMIN_USER=admin
ETHERPAD_DB_HOST=mariadb
ETHERPAD_DB_PORT=3306
ETHERPAD_DB_USER=etheruser
ETHERPAD_DB_PASSWORD=94156BD8hgBjdbjuV2yu
ETHERPAD_DB_NAME=etherpad
ETHERPAD_DB_CHARSET=utf8mb4
```

**Comment by Rillke** [comment]
Another issue was fixed with 51edfc85ab67a4b70f6232fbad9109beff8e131d

The settings.json wasn't recreated.

We also updated some documentation and docker-compose so it's now easier to use.

Still, if you change the DB credentials, you need to connect to the DB and re-do the changes there as well. MariaDB/MySQL Containers do not seem to update the credentials after first setup automatically.

**Comment by Rillke** [comment]
Starting-over again with the current version will be an easier way if no data needs to be persisted than re-doing the changes we did. Thanks for the report.
